query-engine/core/src/query_graph_builder/extractors/filters/composite.rs

```diff
@@ -19,19 +19,19 @@ pub fn parse(input_map: ParsedInputMap, field: &CompositeFieldRef, _reverse: boo
             (filters::IS_SET, input) => Ok(field.is_set(input.try_into()?)),
 
             // To-many composite filters.
-            (filters::EVERY, input) => Ok(field.every(extract_filter(input.try_into()?, &field.typ())?)),
-            (filters::SOME, input) => Ok(field.some(extract_filter(input.try_into()?, &field.typ())?)),
-            (filters::NONE, input) => Ok(field.none(extract_filter(input.try_into()?, &field.typ())?)),
+            (filters::EVERY, input) => Ok(field.every(extract_filter(input.try_into()?, field.typ())?)),
```

Comment:
`typ`! bah.

![image](https://user-images.githubusercontent.com/34967/221205457-de95b4f6-21dc-4d6e-a4e2-48f36478ea3e.png)

Reply:
I always read that as the German "Typ!"